### PR TITLE
update step by step integration with react navigation

### PR DESCRIPTION
**Pull request by TitusEfferian**
## Motivation

when i created fresh project with latest react-native version, and i wanted to integrate react navigation to my project, i followed step by step instruction from http://facebook.github.io/react-native/docs/navigation and it got an error. 

But i found out that the current documentation missed out some step.

here is my pull request give an additional steps with the current react navigation version.

and here is some screenshot of additional step in my changes:

**1. install dependencies**
image

**2. finishing up with android or iOS development target**
image

### CODE REVIEW

**Comment by TitusEfferian** [comment]
hi, can i request review this pull request?
@hramos

**Comment by cpojer** [comment]
@osdnk

**Comment by TitusEfferian** [comment]
hi, i'm still looking forward to hear your suggestion on this PR 🙏 
@cpojer @osdnk

File: website/versioned_docs/version-0.5/navigation.md [hunk 1]
```diff
@@ -12,22 +12,86 @@ If you'd like to achieve a native look and feel on both Android and iOS, or you'
 
 ## React Navigation
 
-The community solution to navigation is a standalone library that allows developers to set up the screens of an app with a few lines of code.
+The community solution to navigation is a standalone library that allows developers to set up the screens of an app with a few lines of code. Assuming you are on the latest react-native version.
```
Comment (osdnk) [inline website/versioned_docs/version-0.5/navigation.md]: Can you rephrase it? React Navigation works with some older versions. What's the point of this sentence? It sounds a bit strange.

File: website/versioned_docs/version-0.5/navigation.md [hunk 2]
```diff
@@ -12,22 +12,86 @@ If you'd like to achieve a native look and feel on both Android and iOS, or you'
...
 
 ```
-npm install --save react-navigation
+yarn add react-navigation
+# or with npm
+# npm install react-navigation
 ```
 
-The second step is to install react-native-gesture-handler
+The second step is to install additional dependencies such as `react-native-gesture-handler`, `react-native-reanimated`, and `react-native-screens`:
```
Comment (osdnk) [inline website/versioned_docs/version-0.5/navigation.md]: and `react-native-safe-area-context` and `masked-view`

File: website/versioned_docs/version-0.5/navigation.md [hunk 3]
```diff
@@ -12,22 +12,86 @@ If you'd like to achieve a native look and feel on both Android and iOS, or you'
...
 ```
 
+```
+yarn add react-native-reanimated
+# or with npm
+# npm install --save react-native-reanimated
+```
+
+```
+yarn add react-native-screens
```
Comment (osdnk) [inline website/versioned_docs/version-0.5/navigation.md]: can you join then into one command like: 
`npm install react-native-reanimated react-native-gesture-handler react-native-screens react-native-safe-area-context @react-native-community/masked-view`.

Also, we in react-navigation are now not supportive of using `yarn`, so If you want, feel free to mention only `npm` (nit).

File: website/versioned_docs/version-0.5/navigation.md (hunk 2 above)
Comment (TitusEfferian) [reply website/versioned_docs/version-0.5/navigation.md]: okay, and alredy joined into one line here -> 571f5ed1056eccac556a608256363587d9993674

File: website/versioned_docs/version-0.5/navigation.md (hunk 1 above)
Comment (TitusEfferian) [reply website/versioned_docs/version-0.5/navigation.md]: sorry i missed this one to remove from my commit 🙏,  it just a local change test because it's my first time trying to contribute here 🙏 

8a25db7d4468a4a82a24348d15d20ff2eeb3c1b2

**Comment by TitusEfferian** [comment]
oops,
i think someone already updated and merged the latest react navigation docs [here](https://github.com/facebook/react-native-website/pull/1578)
